documents/openapi/openapi.yaml

```diff
@@ -1334,6 +1334,58 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /v3/notifications/email:
```

Comment:
There are OpenAPI changes that need to be made here.

Right now, this schema only states that the id_type and id_value are the only required fields beside template_id. That is incorrect. 

**v3 as written**
![Screen Shot 2023-08-30 at 7 49 29 AM](https://github.com/department-of-veterans-affairs/notification-api/assets/111803622/3b4ea6d4-a151-4f65-8d76-a5e6f478b0dd)

- note that anyOf has nothing in it
- note that email_address is not required
- note that recipient_identifier is not required
- note that id_type and id_value are required
- there are two errors in the openapi validator for lines 2339 and 2386
![Screen Shot 2023-08-30 at 7 52 00 AM](https://github.com/department-of-veterans-affairs/notification-api/assets/111803622/b6f2e135-e038-48f7-96aa-ec61d551861b)


**v2 with more accurate description**
Here, anyOf is used correctly - one could use email_address or recipient_identifier or both.
Note too that there are examples for using just email or just recipient_identifier in a dropdown - v3 should also have this and an example for including both
![Screen Shot 2023-08-30 at 7 53 11 AM](https://github.com/department-of-veterans-affairs/notification-api/assets/111803622/88ee2589-a7db-447f-9be3-75825723f463)

Please update.

Reply:
Updated.  Turns out json-schema and Swagger syntaxes are not exactly the same.